tracing/src/main/java/org/axonframework/extensions/tracing/TracingQueryGateway.java

```diff
@@ -8,6 +39,11 @@
 
 import java.util.concurrent.CompletableFuture;
 
+/**
+ * A tracing query gateway which activates a calling {@link Span}, when the {@link CompletableFuture} completes.
+ *
+ * @author Christophe Bouhier
+ */
 public class TracingQueryGateway extends DefaultQueryGateway {
 
     private final Tracer tracer;
```

Comment:
Would it make sense to promote the tracing dispatch and handler interceptors to a global level here?
That would make enforcing those to be a part of the `TracingCommandGateway` easier too, as well as testing their existence in the auto config test.
What do you think?